docs/patched_pdoc.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+def main():
+    from pdoc import cli
+
+    cli.main()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Really curious as in what this is for. Also you are missing the license and `# cython: language_level=3`